app/storage/documents-repo.js

```diff
@@ -3,6 +3,51 @@ const { blobServiceClient } = require('./blob-service-client')
 const { mapMetadataToBlob } = require('../mappers/blob-metadata')
 const config = require('../config/storage')
 
+const getDocuments = async () => {
+  const documentsContainer = blobServiceClient.getContainerClient(config.documentsContainer)
```

Comment:
Not a major thing but you could move this to line 5 rather than recreating it in each function.